wom/src/main/scala/wom/values/WomFile.scala

```diff
@@ -1,67 +1,154 @@
 package wom.values
 
-import wom.types.{WomFileType, WomGlobFileType, WomSingleFileType, WomType}
+import wom.types._
 
 import scala.util.{Success, Try}
 
 sealed trait WomFile extends WomPrimitive {
+
+  // TODO: WOM: WOMFILE: This is often used as _either_ the cloud location _or_ the path within the container. Does this need to be split into two values?
```

Comment:
In a long and distant future something like a `WomPath` could be interesting I think